react/features/base/participants/types.ts

```diff
@@ -8,7 +8,7 @@ export interface Participant {
     e2eeSupported?: boolean;
     email?: string;
     features?: {
-        'screen-sharing'?: boolean;
+        'screen-sharing'?: boolean | string;
```

Comment:
When does it take a string value?

Reply:
It takes a string of a boolean (`"true"`)

Reply:
WAT